src/ShapeCrawler/Drawing/IShapeFill.cs

```diff
@@ -23,6 +23,21 @@ public interface IShapeFill
     /// </summary>
     public string? Color { get; }
 
+    /// <summary>
+    /// Gets the Alpha percentage of the fill color.
+    /// </summary>
+    public double? AlphaPercentage { get; }
+
+    /// <summary>
+    /// Gets the Luminance Modulation Percentage of the fill color.
+    /// </summary>
+    public double? LuminanceModulationPercentage { get; }
+    
+    /// <summary>
+    /// Gets the Luminance Offset Percentage of the fill color.
+    /// </summary>
+    public double? LuminanceOffsetPercentage { get; }
```

Comment:
I would suggest avoiding `null` when it's _possible_. Can we return 100% for Alpha Percentage, 100% for Luminance Modulation Percentage, and 0% for Luminance Offset Percentage instead of `null`?

Reply:
We could but it's less of a reflection of what happens in the Powerpoint file. These fields literally don't exist in the elements when they are null. If we're aiming towards an object representation of the xml, I'd leave it as is, otherwise if we're trying to simplify and save time, we can assume those values: 100% Alpha, 100% Luminance Modulation Percentage and 0% Luminance Offset Percentage

Reply:
I fixed according to this suggestion and also added alpha for when it's a solid color fill

Reply:
Thank you👍

I realize it might be wrong, but I always hope that users will make a bug report with specific case.